### PR TITLE
CCW responsiveness

### PR DESCRIPTION
**Pull request by areyesd14**
In this PR, css were improved to make CCW responsive.

### CODE REVIEW

File: love/src/components/MainTel/CableWraps/CableWraps.module.css [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
     margin: 5px;
     height: 100%; 
-    width: 500px; 
+    width: 100%; 
     margin-top: 0.5em;
 
 }
 
-.cableWrapsContainer .cableWrapsContent {
+/* .cableWrapsContainer .cableWrapsContent {
```
Comment (sebastian-aranda) [inline love/src/components/MainTel/CableWraps/CableWraps.module.css:11]: This should be removed

File: love/src/components/MainTel/CableWraps/CableWraps.module.css [hunk 2]
```diff
@@ -49,14 +49,24 @@
 
 .divCameraWrap {
     margin-top: 2em;
+    display: inline-block;
+    height: 70%;
+    width: 100%;
+	vertical-align: middle;
```
Comment (sebastian-aranda) [inline love/src/components/MainTel/CableWraps/CableWraps.module.css:55]: Are you using pre-commit for linting and formatting? this line should have been reviewed by ESLint and formatted by Prettier

File: love/src/components/MainTel/CableWraps/CameraCableWrap/CameraCableWrap.jsx [hunk 3]
```diff
@@ -36,12 +36,11 @@ class CameraCableWrap extends Component {
   }
 
   createCameraCableWrap(dom) {
-
     const { width, height } = this.props;
 
     const radio = height - 60;
-
-    const svg = d3.select(dom).append('svg').attr('class', 'd3').attr('width', width).attr('height', height);
+    // .attr('width', width).attr('height', height)
```
Comment (sebastian-aranda) [inline love/src/components/MainTel/CableWraps/CameraCableWrap/CameraCableWrap.jsx:42]: This should be removed

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Good work @areyesd14 ! thanks. I've left some minor comments that should be checked before merge

File: love/src/components/MainTel/CableWraps/CableWraps.module.css (hunk 1 above)
Comment (areyesd14) [reply love/src/components/MainTel/CableWraps/CableWraps.module.css]: Removed! ;)

File: love/src/components/MainTel/CableWraps/CameraCableWrap/CameraCableWrap.jsx (hunk 3 above)
Comment (areyesd14) [reply love/src/components/MainTel/CableWraps/CameraCableWrap/CameraCableWrap.jsx]: Removed!

**Review by sebastian-aranda — APPROVED** [review]
Thanks @areyesd14 !

**Pull request merged**